File: src/main/java/me/santio/uhc/sidebars/LobbySidebar.java
package me.santio.uhc.sidebars;

import me.santio.uhc.UHC;

public class LobbySidebar extends Sidebar {
    
    public LobbySidebar() {
        super("lobby", UHC.getMainColor()+"UHC &8| &7Lobby");
        this.setLine(1, "Test :)");
        this.setLine(3, "Hey!");
    }

}

File: src/main/kotlin/me/santio/uhc/sidebars/LobbySidebar.kt
package me.santio.uhc.sidebars

import me.santio.uhc.UHC

class LobbySidebar : Sidebar("lobby", "${UHC.mainColor}UHC &8| &7Lobby") {
    init {
        setLine(1, "Test :)")
        setLine(3, "Hey!")
    }
}